base/strings/unicode.jl

```diff
@@ -33,7 +33,8 @@ isvalid(value)
 
 Returns `true` if the given value is valid for that type. Types currently can
 be either `AbstractChar` or `String`. Values for `AbstractChar` can be of type `AbstractChar` or [`UInt32`](@ref).
-Values for `String` can be of that type, or `Vector{UInt8}` or `SubString{String}`.
+Values for `String` can be of that type, `SubString{String}`, `Vector{UInt8}`,
+or a contiguous subarray thereof.
```

Comment:
Perhaps this needs a `compat` annotation that it requires 1.6?

Reply:
Done.